web/Gruntfile.js

```diff
@@ -100,6 +101,9 @@ module.exports = function (grunt) {
         }
     });
 
+    const karmaTask = require('./grunt/karma-config')(grunt);
```

Comment:
Now that `karma-config` is a grunt plugin, you can `grunt.loadTasks('./grunt')`, which means any number of grunt plugins can be added to that directory and they'll all be loaded.